actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -2246,6 +2246,8 @@ def button(value = nil, options = {}, &block)
         @template.button_tag(value, options, &block)
       end
 
+      # Returns true if the model id field has been added to a hidden form field.
+      #
       def emitted_hidden_id?
```

Comment:
This method should be `# :nodoc:` It is not part of the public API.

Reply:
Ok, but if this method is not part of the public API, it should be a private method, what you think? Because it's not be used in other part of this project.